### PR TITLE
Create 2020-06-07-coreweekly-27-2020.md

### PR DESCRIPTION
**Pull request by ttoine**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Please be specific when describing the PR. Every detail helps: What are you changing? Why?
| Type?         | bug fix / improvement / new feature / refacto
| BC breaks?    | yes / no
| Deprecations? | yes / no
| Fixed ticket? | Fixes {paste the issue here}.
| How to test?  | Please indicate how to best verify that this PR is correct.

### CODE REVIEW

**Comment by matks** [comment]
No mention of 1.7.6.6 nor 1.7.6.7 in "General messages" ? 😉

**Review by Progi1984 — APPROVED** [review]

File: news/_posts/core-weekly/2020-06-07-coreweekly-27-2020.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+image: /assets/images/2017/04/core_weekly_banner.jpg
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 29th of June to Sunday 5th of July 2020.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
```
Comment (ttoine) [inline news/_posts/core-weekly/2020-06-07-coreweekly-27-2020.md:16]:
```suggestion

## General messages

PrestaShop [1.7.6.6](https://build.prestashop.com/news/prestashop-1-7-6-6-maintenance-release/) and [1.7.6.7](https://build.prestashop.com/news/prestashop-1-7-6-7-maintenance-release/) have been released, don't forget to update as they fix security issues and regressions.

```

File: news/_posts/core-weekly/2020-06-07-coreweekly-27-2020.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 27 of 2020"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2020-06-07
```
Comment (matks) [inline news/_posts/core-weekly/2020-06-07-coreweekly-27-2020.md]:
```suggestion
date:   2020-06-08
```

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @ttoine

**Comment by matks** [comment]
Although this PR was merged, the article is not published on https://build.prestashop.com/

**Comment by matks** [comment]
> Although this PR was merged, the article is not published on https://build.prestashop.com/

Understood, there was a mistake in the filename and the publication date

=> fix https://github.com/PrestaShop/prestashop.github.io/pull/742

**Pull request merged**